config/zones/CH.yaml

```diff
@@ -342,22 +342,22 @@ isRenewable:
   unknown:
     - datetime: '2017-01-01'
       source: SFOE, Electricity Maps
-      value: 0.53
+      value: 0.72
```

Comment:
This is quite a large change, do you know why that might be?

Reply:
Yes, in the source file, I think we treated the total unknown **thermal** power production (which includes non-renewable as well as renewable = biomass sources) as non-renewable energy source. Since biomass makes up a significant share of this and has a much lower emission factor, this led to the decrease in emission factors (and renewable and low carbon shares). Happy to jump on a brief call if you'd like to review these findings 😃

Reply:
Ah then it makes perfect sense!
No need for a call as this is not the first time this has happened 😅.